walkingpadfitbit/domain/eventhandler.py

```diff
@@ -35,15 +37,33 @@ def handle_treadmill_event(self, event: TreadmillEvent):
         else:
             self._on_walk(event)
 
+    async def _get_daily_summary(self):
+        self._daily_summary = await self._remote_activity_repository.get_daily_summary()
+        logger.info(f"Daily summary is {self._daily_summary}")
+
     def _on_walk(self, event: TreadmillEvent):
-        self._print_event_output(event_text=self.display.walk_event_to_text(event))
+        self._print_event_output(
+            event_text=self.display.walk_event_to_text(
+                event=event,
+                daily_summary=self._daily_summary,
+            )
+        )
+        if not self._last_walk_event:
```

Comment:
Kick off the fetch before printing the data

Reply:
Also handle the cases of:

* a successful response from Fitbit (including no daily activity yet): show the calculated totals
* no data retrieved from Fitbit (no response yet or error): show dashes for totals